doc/src_usermanual/techrep_main.tex

```diff
@@ -784,45 +784,210 @@ \section{Solver options}
+
+\medskip
+
+
+\noindent \textbf{duals\_init}: type of the update for the initialization of Lagrange multipliers corresponding to  the equality constraints. Possible values one of the the strings "lsq" (least-square (LSQ) solve initialization) and "zero"(multipliers are set identically to zero). Default value is "lsq". 
+
+\medskip
+
+\noindent \textbf{duals\_lsq\_ini\_max}: max inf-norm allowed for initial duals when computed with LSQ (see \textbf{duals\_init}); if norm is greater, the duals for the equality constraints will be set to zero. Double values between $10^{-16}$ and $10^{10}$. Default value: $1\,000$.
+
+\medskip
+
+%%%%
+
+\noindent \textbf{duals\_update\_type}: string option specifying the type of update of the multipliers of the eq. constraints after each iteration. Possible values are ``lsq'' (update based on a LSQ solve) and ``linear'' (Newton update based on the dual steplength. When ``Hessian'' is ``quasinewton\_approx'' the default value for this options is ``lsq''. When ``Hessian'' is ``analytical\_exact'' the default value is ``linear''.
+
+\medskip
+
+\noindent \textbf{recalc\_lsq\_duals\_tol}: threshold for inf-norm under which the LSQ computation of duals is used. If the inf-norm of the duals of the equality constraints is larger than the value of this options, these duals are set to zero. This options requires ``duals\_update\_type'' to be ``lsq'' (the option is ignored otherwise). Double values in $[0,10^{10}]$. Default value $10^{-6}$.
 
 
+\medskip
+
+\noindent \textbf{accept\_every\_trial\_stepduals}: disable the line-search and take the close-to-boundary step. String values: ``no'' (default) and ``yes''.
+
+\medskip
+
+\noindent \textbf{Hessian}: type of Hessian used with the filter IPM.
+\begin{itemize}
+\item ``quasinewton\_approx`` (default) - HiOp will build secant BFGS approximation for the Hessian and use a quasi-Newton filter IPM
+\item ``analytical\_exact'' - Hessian provided by the user and a Newton filter IPM algorithm will be used
+\end{itemize}
+
+\medskip
+
+\noindent \textbf{sigma0}: initial value of the initial multiplier of the identity in the secant approximation. Numeric values in $[0,10^7]$. Default value $1$.
+
+\medskip
+
+\noindent \textbf{secant\_memory\_len}: size of the memory (number of $(s,y)$ pairs) of the Hessian secant approximation. Integer values between $0$ and $256$. Default value $6$.
+
+\subsection{Linear algebra computational kernels}
+
+\noindent \textbf{KKTLinsys}: type of KKT linear system \textit{formulation} used internally
+\begin{itemize}
+\item ``auto'' (default): decided by \Hi based on the type of interface/NLP solved and ``compute\_mode`` and ``Hessian`` options
+\item ``xycyd'': symmetric indefinite (less stable but smaller size)
+\item ``xdycyd'': symmetric indefinite (more stable but  larger size)
+\item  ``full'': unsymetric suitable for LU solvers (experimental)

```

Comment:
should we mention this option in this release?

Reply:
it says is experimental ;) let's leave it there, we may forget to add it later on.